trumania/core/relationship.py

```diff
@@ -11,15 +11,15 @@
 
 # There are a lot of somewhat ugly optimizations here like in-place mutations,
 # caching, or usage of numpy instead of a more readable pandas alternative. The
-# reason is the methods of this filetend to be called a large amount of time
+# reason is the methods of this file tend to be called a large amount of time
 # in inner loop of the simulation, optimizing them make the whole simulation
 # faster.
 
 
-class Relations(object):
+class OutgoingRelationships(object):
```

Comment:
I like the addition of `Outgoing`, that indeed clarifies the direction. 
However I would not use the term `relationship` in this class since that term has a specific meaning in the context of Trumania => I think that would cause a confusion between `Relationship` and `OutgoingRelationship` (maybe suggesting the later is a specific case of the former)

=> how about `OutgoingRelations` ?